example-mojaloop-backend/values.yaml

```diff
@@ -214,43 +105,42 @@ account-lookup-mysql:
     ##   replicator: /vault/secrets/mysql-replicator
     ##
     customPasswordFiles: {}
-  primary:
-    persistence:
-      enabled: false
-
-    ## @param secondary.affinity Affinity for MySQL secondary pods assignment
-    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-    ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
-    ##
-    affinity: {}
-
-    ## @param secondary.nodeSelector Node labels for MySQL secondary pods assignment
-    ## ref: https://kubernetes.io/docs/user-guide/node-selection/
-    ##
-    # nodeSelector: {}
-    
-    ## Example
-    # nodeSelector:
-    #     "node-role.mojaloop.io": als
-
-    ## @param secondary.tolerations Tolerations for MySQL secondary pods assignment
-    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
-    ##
-    # tolerations: []
-    
-    ## Example
-    # tolerations:
-    #   - key: "node-role.mojaloop.io"
-    #     operator: "Equal"
-    #     value: als
-    #     effect: "NoSchedule"
-
-  secondary:
-    persistence:
-      enabled: false
+  ## @param initdbScripts Dictionary of initdb scripts
+  ## Specify dictionary of scripts to be run at first boot
+  initdbScripts:
+    # This script enables legacy authentication for MySQL v8. NodeJS MySQL Client currently does not support authentication plugins, reference: https://github.com/mysqljs/mysql/pull/2233
+    # Note : also initialises mysql DBs to support thirdparty charts
+    accountLookupInit.sql: |-
+      DROP DATABASE IF EXISTS account_lookup;
+      CREATE DATABASE account_lookup;
+      DROP USER IF EXISTS account_lookup@'%';
+      CREATE USER 'account_lookup'@'%' IDENTIFIED WITH mysql_native_password BY 'HY#Mz%_z';
```

Comment:
Is there no way to access the passsword through an env var instead of hard-coding this?

Applies to the other scripts below.

Reply:
@mdebarros : Yeah there is and I think we were putting in (in fact we already have issues for this they are #2720 and #3082